src/Components/Registration/registrationForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Button, Form, Spinner } from "react-bootstrap";
+import "./registrationForm.css";
+
+import { RegistationRequest } from "../../Redux/createAction/createAction";
+import { useHistory } from "react-router-dom";
+
+const Registration = () => {
+  const [username, setusername] = useState("");
+  const [password, setpassword] = useState("");
+  const [option, setoptions] = useState("Admin");
+  const history = useHistory();
+
+  const dispatch = useDispatch();
+  const state = useSelector((state) => {
+    return state;
+  });
+
+  const regisstatus = state.Regis_status;
+  const handlesubmit = () => {
+    let formData = {
+      username: username,
+      password: password,
+      option: option,
+    };
+    dispatch(RegistationRequest(formData));
+    setusername("");
+    setpassword("");
+    if (regisstatus.isLoading === false && regisstatus.isRegistered === false) {
+    }
+  };
+  if (regisstatus.isRegistered) {
```

Comment:
??

Reply:
do we need to redirect on login page here ?

Reply:
i am using functional component,where there is no render() method....therefore i am using {useSelector} hooks to get the state from the store and redirecting when "isRegistered" get true

Reply:
i am asking why we need to redirect on login page , from register page

Reply:
user is redirecting to Login page after when user get successfully registered.
do we not need it here?

Reply:
not needed , there is a button after register, show successfully registered and there is button back to login
on click you will not redirect to login , you will push to login component

Reply:
ok